src/lib/Form/DataTransformer/LocationsTransformer.php

```diff
@@ -27,19 +28,41 @@ public function __construct(LocationService $locationService)
         $this->locationService = $locationService;
     }
 
+    /**
+     * Transforms a domain specific Location objects into a Location's ID comma separated string.
+     *
+     * @param mixed $value
+     * @return array|mixed|string
+     */
     public function transform($value)
     {
-        return is_array($value) && !empty($value)
-            ? implode(',', array_column($value, 'id'))
-            : [];
+        /** TODO add sanity check is array of Location object? */
+        if (!is_array($value) || empty($value)) {
+            return [];
+        }
+
+        return implode(',', array_column($value, 'id'));
     }
 
-    public function reverseTransform($value)
+    /**
+     * Transforms a Location's ID string into a domain specific Location objects.
+     *
+     * @param mixed $value
+     * @return array|null
```

Comment:
Maybe better would be `@return Location[]|null` if we know the type of elements in returning collection?